java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java

```diff
@@ -3369,6 +3377,16 @@ public MatchType isNumberMatch(PhoneNumber firstNumberIn, PhoneNumber secondNumb
     return MatchType.NO_MATCH;
   }
 
+  // Return true if extension matches, possibly ignoring control characters
```

Comment:
I'd remove "possibly" because we are ignoring them (whether or not they're present).